robot/web/web.go

```diff
@@ -176,7 +176,7 @@ func allAudioSourcesToDisplay(theRobot robot.Robot) map[string]gostream.AudioSou
 			continue
 		}
 
-		sources[name] = input
+		sources[name] = audioinput.NewGostreamWrapper(input)
```

Comment:
This essentially strips away the extra params before feeding AudioInput to the streamer. So if a user's custom AudioInput needed those extra params, streaming wont work on the app page.

Reply:
can you add a comment here?